features/custom_matchers/define_matcher.feature

```diff
@@ -335,3 +335,109 @@ Feature: define matcher
       | 4 examples, 2 failures               |
       | expected 9 to be a multiple of 2     |
       | expected 9 not to be a multiple of 3 |
+
+  Scenario: compare values
+
+    `RSpec` comes with some handy helper methods. One is `values_match?` which
+    is available if you define your matcher using the "matcher"-DSL.
+
+    The order of arguments is important for `values_match?`, e.g. especially if
+    your matcher should handle `Regexp`-objects (`/regex/`): First comes the
+    `expected` value, second the `actual` one.
+
+    This method also supports matchers as `expected` value.
+
+    Given a file named "compare_values_spec.rb" with:
+      """ruby
+      RSpec::Matchers.define :have_content do |expected|
+        match do |actual|
+          values_match? expected, actual
+        end
+      end
+
+      RSpec.describe 'a' do
+        it { is_expected.to have_content 'a' }
+      end
+
+      RSpec.describe 'a' do
+        it { is_expected.to have_content /a/ }
+      end
+
+      RSpec.describe 'a' do
+        it { is_expected.to have_content a_string_starting_with('a') }
+      end
+      """
+    When I run `rspec ./compare_values_spec.rb --format documentation`
+    Then the exit status should be 0
+
+  Scenario: Error handling
+
+    Make sure your matchers returns either `true` or `false`. Take care to handle
+    exceptions appropriately in your matcher, e.g. most cases you might want
+    your matcher to return `false` if an exception - e.g. ArgumentError -
+    occures, but there might be edge cases where you want to pass the exception
+    to the user.
```

Comment:
It was part of the preamble in one of my first versions. But I thought this might move users in the wrong direction and litters the preamble.